src/read/pe/pe_export.rs

```diff
@@ -0,0 +1,97 @@
+use core::fmt::Debug;
+
+/// Possible exports from a PE file
+#[derive(Clone)]
+pub enum PeExport<'data> {
```

Comment:
Maybe split this in an enum describing what (name or ordinal of the export) and one describing where (address or forwarded symbol)?

Reply:
I've renamed to `pe::Export` and renamed the file.

I'm not sure why the proposed struct would be a better choice however. (also, exports can be forwarded _and_ have a name, so we'd need at least a second `Option<&[u8]>`, and a forwarded export has no address, so `address` should be `Option`al as well).

To sum up, every export has an ordinal. Some can have names. And they are either local or forwarded, so the most sensible solution would be something like
```rust
pub struct Export {
    pub ordinal: u32,
    pub name: Option<&[u8]>,
    pub target: Enum {
        Local(u64),
        Forwarded(&[u8]),
    }
}
```
(that's not correct Rust but you get the idea)

Would you find this more handy as what I have currently proposed?

Reply:
Yes, that looks good. The reason I want this change is that it better matches the underlying data (name and address are separate arrays) and I believe it will make the code simpler.

Reply:
I've changed this struct and pushed